src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -11,28 +11,28 @@
 import seedu.address.commons.exceptions.IllegalValueException;
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.lab.Lab;
+import seedu.address.model.student.Student;
+import seedu.address.model.student.lab.Lab;
 
 /**
  * An Immutable TAddressBook that is serializable to JSON format.
  */
 @JsonRootName(value = "addressbook")
```

Comment:
Do we want to change this to "TAddressbook". Could also leave this for now since this PR's primary focus is to change `Person` to `Student`.